src/pcm.rs

```diff
@@ -949,6 +931,35 @@ impl Status {
     }
 }
 
+/// Builder for [`Status`].
+///
+/// Allows setting the audio timestamp configuration before retrieving the
+/// status from the stream.
+pub struct StatusBuilder(Status);
+
+impl StatusBuilder {
+    pub fn new() -> Self {
+        StatusBuilder(Status::new())
+    }
+
+    pub fn set_audio_htstamp_config(
+        self,
```

Comment:
Builder pattern shouldn't have "set_" in its name. Probably just "audio_htstamp_config" is the right name here.